README.md

```diff
@@ -111,6 +111,10 @@ When running on cluster node, you will also need to distribute this keytab, belo
 
 When Spark application is started, it will transparently track the execution plan of submitted SQL/DF transformations, parse the plan and create related entities in Atlas.
 
+Reference
+===
+- Mingjie Tang, Saisai Shao, Weiqing Yang, Yanbo Liang, Yongyang Yu, Bikas Saha, Dongjoon Hyun. [SAC: A System for Big Data Lineage Tracking](http://merlintang.github.io/paper/sac_icde.pdf). In IEEE 35th International Conference on Data Engineering (ICDE), 2019
```

Comment:
Rather than referring an individual github link why not just have the pdf as a part of the SAC repo itself?

Instead of a separate section we could just have it listed under the "Spark Atlas Connector" section in the README. Just a line as below with the link.
[SAC: A System for Big Data Lineage Tracking](url)

Reply:
I wouldn't include anything unless it's safe to claim that it's under copyright of Cloudera, or the license of paper is clear to be compatible with ASF.

Reply:
it is removed

Reply:
No I meant link to external would be OK unless restricted. It's a different story if we "include" the paper in repo, so I pointed out for that.